pages/class/[classid].tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useContext } from "react";
+import type { NextApplicationPage } from "../_app";
+import { AuthContext } from "../../context/AuthContext";
+import styles from "../../styles/class.module.css";
+import { Error } from "../../components/util/Error";
+import { Roster } from "../../components/Roster/Roster";
+import { useRouter } from "next/router";
+import { Attendance } from "../../components/Attendance/Attendance";
+import { Module } from "../../components/ModuleView/Module";
+import useSWR from "swr";
+import { APIContext } from "../../context/APIContext";
+
+const BackButton: React.FC = () => {
```

Comment:
also, since we will be using this back button a lot, can we put it into the util folder? Have it take a prop called `linkTo: string` that will pass in the route we need to go to when its clicked. Then, all the other people can use this